engine/src/main/resources/templates/Inventory/validationErrorEmailBody.html

```diff
@@ -0,0 +1,40 @@
+{#include Inventory/insightsEmailBody}
+{#content-title}{action.context.event_name} Instant Notification - {action.timestamp.toStringFormat()}{/content-title}
```

Comment:
this event name is the one I am sending in context, it is supposed to be "Host Validation Error"

Reply:
ah - nevermind here, confused it with action.event_type :D